src/geo/transform.js

```diff
@@ -398,10 +401,11 @@ class Transform {
      * Calculate the posMatrix that, given a tile coordinate, would be used to display the tile on a map.
      * @param {UnwrappedTileID} unwrappedTileID;
      */
-    calculatePosMatrix(unwrappedTileID: UnwrappedTileID): Float32Array {
+    calculatePosMatrix(unwrappedTileID: UnwrappedTileID, aligned: boolean = false): Float32Array {
         const posMatrixKey = unwrappedTileID.key;
-        if (this._posMatrixCache[posMatrixKey]) {
-            return this._posMatrixCache[posMatrixKey];
+        const cache = aligned ? this._alignedPosMatrixCache : this._posMatrixCache;
+        if (!aligned && cache[posMatrixKey]) {
```

Comment:
🤔 would we not want to use the `alignedPosMatrixCache` result if it's available?